com.ktechpit.wonderwall.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: com.ktechpit.wonderwall
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.12'
+runtime: org.kde.Platform
+runtime-version: '5.12'
```

Comment:
5.15 I said.

Reply:
I tried building with Qt5.15 but the build require many changes in source code, which is due to Qt's migration step to version 6. My app was built with Qt5.5.